bedrock/privacy/views.py

```diff
@@ -70,8 +70,18 @@ def get_legal_doc(self):
         return doc
 
 
-firefox_notices = PrivacyDocView.as_view(
-    template_name='privacy/notices/firefox-quantum.html',
+class FirefoxPrivacyDocView(LegalDocView):
```

Comment:
I think I would add a default to the class itself, then override that in the method if the selector matches:

```python
class FirefoxPrivacyDocView(LegalDocView):
    template_name = 'privacy/notices/firefox.html'
    ...
```
Then just remove that else statement in the `get_legal_doc` method.

Reply:
Also just subclass `PrivacyDocView` and remove line 76.